etc/kayobe/grafana.yml

```diff
@@ -0,0 +1,58 @@
+---
+###############################################################################
+# Grafana configuration.
+
+# Grafana local admin user name. If you are deploying Monasca Grafana this
+# should not conflict with an OpenStack user name.
+#grafana_local_admin_user_name:
```

Comment:
If you go with my suggestion of using admin as the default in kayobe, this will need to be overridden.

Reply:
done